docs/man/sat-bootsys.8.rst

```diff
@@ -17,57 +17,87 @@ SYNOPSIS
+shut down the compute nodes and User Access Nodes (UANs). It attempts to check
+whether the nodes are already in an "Off" state in the Hardware State Manager
+before attempting to shut them down with BOS.
+
+In the ``cabinet-power`` stage, it suspends the ``hms-discovery`` Kubernetes
+cron job and uses the Cray Advanced Platform Monitoring and Control (CAPMC)
+service to power off all liquid-cooled and air-cooled compute node cabinet and
+non-management NCN cabinets. It then waits for all components in those cabinets
+to reach the power "Off" state.
+
+In the ``platform-services`` stage, it stops all management services on the
+non-compute nodes (NCNs). This includes backing up and stopping the etcd
+cluster, stopping the kubelet service on each node, freezing ceph, and stopping
+all containers running in containerd. This ansible playbook is known to
+experience a problem that results in it hanging when stopping containers in
+containerd, so if this occurs, a workaround will automatically be executed that
+kills hanging processes on each node where it is necessary. Once the playbook
+has completed, it will move on to the next stage.
+
+In the ``ncn-power`` stage, it shuts down Linux on all NCNs, and powers them off
+using IPMI. Once this has completed, and it is safe to shutdown and power off
+ncn-m001 where this command was run.
 
 BOOT ACTION
 -----------
 
-The boot action is not currently implemented and will exit with a non-zero exit
-code and an error message.
+The boot action consists of the following stages. Similar to the shutdown
+action, each stage automates a specific portion of the system boot procedure.
+
+In the ``ncn-power`` stage, ``sat bootsys`` powers on all mangagement NCNs and
+boots Linux, and waits for the management cluster to become accessible over the
+network.
+
+In the ``platform-services`` stage, it ensures that conatinerd and etcd are are
+running and enabled on all Kubernetes NCNs, then starts Ceph services, unfreezes
+Ceph, and waits for Ceph to become healthy. After this, it starts and enables
+kubelet on all Kubernetes NCNs.
+
+In the ``k8s-check`` stage, it waits for the Kubernetes cluster to become
+available and for the Kuberenetes pods to become healthy. Specifically, it waits
+for each pod that was running prior to shutdown to have a similarly-named pod
+after boot, and any new pods are expected to be in the "Running" or "Succeeded"
+states.
+
+In the ``cabinet-power`` stage, it enables the ``hms-discovery`` Kubernetes cron
+job and uses the Cray Advanced Platform Monitoring and Control (CAPMC) service
+to power on all liquid-cooled and air-cooled compute node cabinets and
+non-management NCN cabinets. It then waits for all components in those cabinets
+to reach the power "On" state.
+
+In the ``bos-operations`` stage, it uses the Boot Orchestration Service (BOS) to
+boot the compute nodes and User Access Nodes (UANs). It attempts to check
```

Comment:
Could use similar wording here about shutting down whatever nodes are specified in the provided session templates.